tests/e2e/tests/critical-path/browser/filtering.e2e.ts

```diff
@@ -1,69 +1,66 @@
-import { addNewStandaloneDatabase } from '../../../helpers/database';
+import { acceptLicenseTermsAndAddDatabase, deleteDatabase } from '../../../helpers/database';
 import {
-    MyRedisDatabasePage,
     BrowserPage,
-    UserAgreementPage,
-    CliPage,
-    AddRedisDatabasePage
+    CliPage
 } from '../../../pageObjects';
 import {
     commonUrl,
     ossStandaloneConfig
 } from '../../../helpers/conf';
 import { COMMANDS_TO_CREATE_KEY, KeyTypesTexts } from '../../../helpers/constants';
 import { keyTypes } from '../../../helpers/keys';
+import { Chance } from 'chance';
 
-const myRedisDatabasePage = new MyRedisDatabasePage();
 const browserPage = new BrowserPage();
-const userAgreementPage = new UserAgreementPage();
-const addRedisDatabasePage = new AddRedisDatabasePage();
 const cliPage = new CliPage();
+const chance = new Chance();
+
+let keyName = chance.word({ length: 10 });
 
 fixture `Filtering per key name in Browser page`
     .meta({type: 'critical_path'})
     .page(commonUrl)
-    .beforeEach(async t => {
-        await t.maximizeWindow();
-        await userAgreementPage.acceptLicenseTerms();
-        await t.expect(addRedisDatabasePage.addDatabaseButton.exists).ok('The add redis database view', {timeout: 20000});
-        await addNewStandaloneDatabase(ossStandaloneConfig);
+    .beforeEach(async () => {
+        await acceptLicenseTermsAndAddDatabase(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+    })
+test
+    .after(async () => {
+        //Clear and delete database
+        await browserPage.deleteKeyByName(keyName);
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    })
+    ('Verify that user can search a key with selected data type is filters', async t => {
+        keyName = chance.word({ length: 10 });
```

Comment:
I see it's already defined on line 18, why need to do that again here?

Reply:
the same as dbNames, I need to have different keys to remove them after test execution and don't have the same key in other tests.